### DISCUSSION TITLE
Google Script

### DISCUSSION THREAD

**Discussion by hannydevelop** [Byron/google-apis-rs#398] [Q&A] [answered]
Hi @Byron, thank you for this great work.
I wanted to run the execution script, then I realised that you stopped the development of `https://crates.io/crates/google-script1`. Is there any alternative?

**Comment by Byron** [comment, marked as answer]
The `script1` is ignored because code for it can't be generated. It (somehow) has a duplicate type definition or variant named `Feed`.
Maybe alternatives are on `crates.io`, alternatively one can make calls to the API using `reqwest` for example.

**Reply by hannydevelop** [reply]
> The `script1` is ignored because code for it can't be generated. It (somehow) has a duplicate type definition or variant named `Feed`.
> Maybe alternatives are on `crates.io`, alternatively one can make calls to the API using `reqwest` for example.

I'll try this and update this thread

**Comment by hannydevelop** [comment]
Hi @Byron I just wanted to update that `post` request with `reqwest` and `yup_oauth2` was successful. If it's something you'd like, I can work on a PR, for documentation so some other person can find it easier to integrate theirs.

**Reply by Byron** [reply]
That's great to hear, and thanks for the offer. You could write it up in a gist and link it here so it's related to this original post. A PR seems unnecessary since no code change is done.
